docs/core/install/linux-ubuntu.md

```diff
@@ -38,30 +37,44 @@ The following table is a list of currently supported .NET releases and the versi
 
 ## 22.04
 
-[!INCLUDE [linux-prep-intro-apt](includes/linux-prep-intro-apt.md)]
+> Warning: If you've previously installed .NET 6 from `packages.microsoft.com`, see the [Advisory on installing .NET 6 on Ubuntu 22.04].
```

Comment:
You don't actually have a link here. Please review the [syntax for alerts](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#note-warningtipimportant) (warning/tip/note/etc) and fix up this one and the one below.